README.md

```diff
@@ -16,3 +14,37 @@ To dive into the vast world of Autoware and understand how Autoware Universe fit
 ### Explore Autoware Universe documentation
 
 For those looking to explore the specifics of Autoware Universe components, the [Autoware Universe Documentation](https://autowarefoundation.github.io/autoware.universe/), deployed with MKDocs, offers detailed insights.
+
+## Code Coverage Metrics
+
+Below table shows the coverage rate of entire Autoware Universe and sub-components respectively.
+
+### Entire Project Coverage
+
+[![codecov](https://codecov.io/github/autowarefoundation/autoware.universe/graph/badge.svg?token=KQP68YQ65D)](https://codecov.io/github/autowarefoundation/autoware.universe)
+
+### Component-wise Coverage
+
+You can check more details by clicking the badge and navigating the codecov website.
+
+| Component    | Coverage                                                                                                                                                                                                                                                                                                        |
+| ------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Common       | [![codecov](https://img.shields.io/badge/dynamic/json?url=https://codecov.io/api/v2/github/autowarefoundation/repos/autoware.universe/components&label=Common%20Packages&query=$.[0].coverage)](https://app.codecov.io/gh/autowarefoundation/autoware.universe?components%5B0%5D=Common%20Packages)             |
+| Control      | [![codecov](https://img.shields.io/badge/dynamic/json?url=https://codecov.io/api/v2/github/autowarefoundation/repos/autoware.universe/components&label=Control%20Packages&query=$.[1].coverage)](https://app.codecov.io/gh/autowarefoundation/autoware.universe?components%5B0%5D=Control%20Packages)           |
+| Evaluator    | [![codecov](https://img.shields.io/badge/dynamic/json?url=https://codecov.io/api/v2/github/autowarefoundation/repos/autoware.universe/components&label=Evaluator%20Packages&query=$.[2].coverage)](https://app.codecov.io/gh/autowarefoundation/autoware.universe?components%5B0%5D=Evaluator%20Packages)       |
+| Launch       | [![codecov](https://img.shields.io/badge/dynamic/json?url=https://codecov.io/api/v2/github/autowarefoundation/repos/autoware.universe/components&label=Launch%20Packages&query=$.[3].coverage)](https://app.codecov.io/gh/autowarefoundation/autoware.universe?components%5B0%5D=Launch%20Packages)             |
```

Comment:
It looks like Launch is not showing any percentage maybe because the launch packages only contains XML without any test codes. Do you think it still make sense to include this in the table? I though we could add it when we think that there is a need to show the coverage.
![image](https://github.com/user-attachments/assets/404e64fa-ee2c-450f-bfe3-6dd2f119dee7)

Reply:
I think launch is worth tracking the coverage because we should check if specified modules are running given the arguments.